--- xfsdump-2.2.13/m4/manual_format.m4.orig	2003-06-25 03:51:34.000000000 +0200
+++ xfsdump-2.2.13/m4/manual_format.m4	2003-08-08 21:33:33.000000000 +0200
@@ -6,12 +6,5 @@
 # 
 AC_DEFUN([AC_MANUAL_FORMAT],
   [ have_zipped_manpages=false
-    for d in ${prefix}/share/man ${prefix}/man ; do
-        if test -f $d/man1/man.1.gz
-        then
-            have_zipped_manpages=true
-            break
-        fi
-    done
     AC_SUBST(have_zipped_manpages)
   ])
--- xfsdump-2.2.13/m4/package_utilies.m4.orig	2003-06-25 03:51:34.000000000 +0200
+++ xfsdump-2.2.13/m4/package_utilies.m4	2003-08-08 21:33:42.000000000 +0200
@@ -17,9 +17,7 @@
 #  MSGFMT MSGMERGE RPM
 #
 AC_DEFUN([AC_PACKAGE_UTILITIES],
-  [ if test -z "$CC"; then
-        AC_PROG_CC
-    fi
+  [ AC_PROG_CC
     cc="$CC"
     AC_SUBST(cc)
     AC_PACKAGE_NEED_UTILITY($1, "$cc", cc, [C compiler])
