--- xfsdump-2.1.3/configure.in.orig	Thu Aug 15 13:43:40 2002
+++ xfsdump-2.1.3/configure.in	Thu Aug 15 13:45:51 2002
@@ -156,7 +156,7 @@
 savedCFLAGS="$CFLAGS"
 
 dnl Checks for curses header and library.
-AC_CHECK_HEADER(curses.h,, [
+AC_CHECK_HEADER(ncurses/curses.h,, [
 	echo
 	echo 'FATAL ERROR: could not find a valid curses header.'
 	exit 1
@@ -293,13 +293,6 @@
 dnl also check if man page source is gzipped
 dnl (usually on Debian, but not Redhat pre-7.0)
 have_zipped_manpages=false
-for d in ${prefix}/share/man ${prefix}/man ; do
-    if test -f $d/man1/man.1.gz
-    then
-	have_zipped_manpages=true
-	break
-    fi
-done
 AC_SUBST(have_zipped_manpages)
 
 dnl build definitions for use in Makefiles
