--- xfsdump-2.2.13/m4/manual_format.m4.orig	2003-06-25 03:51:34.000000000 +0200
+++ xfsdump-2.2.13/m4/manual_format.m4	2003-08-08 21:33:33.000000000 +0200
@@ -6,12 +6,5 @@
 # 
 AC_DEFUN([AC_MANUAL_FORMAT],
   [ have_zipped_manpages=false
-    for d in ${prefix}/share/man ${prefix}/man ; do
-        if test -f $d/man1/man.1.gz
-        then
-            have_zipped_manpages=true
-            break
-        fi
-    done
     AC_SUBST(have_zipped_manpages)
   ])
--- xfsdump-3.1.9/dump/Makefile.orig	2020-05-06 19:15:10.923504544 +0200
+++ xfsdump-3.1.9/dump/Makefile	2020-05-06 20:53:02.985026191 +0200
@@ -100,9 +100,6 @@
 	$(INSTALL) -m 755 -d $(PKG_ROOT_SBIN_DIR)
 	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_ROOT_SBIN_DIR)
 	$(INSTALL) -m 755 -d $(PKG_SBIN_DIR)
-	# skip symlink when /sbin is alread symlinked to /usr/sbin, like on Fedora
-	test $(PKG_ROOT_SBIN_DIR) -ef $(PKG_SBIN_DIR) || \
-		$(INSTALL) -S $(PKG_ROOT_SBIN_DIR)/$(LTCOMMAND) $(PKG_SBIN_DIR)/$(LTCOMMAND)
 install-dev:
 
 .dep: $(COMMINCL) $(COMMON) $(INVINCL) $(INVCOMMON)
--- xfsdump-3.1.9/restore/Makefile.orig	2020-05-06 19:15:10.926837859 +0200
+++ xfsdump-3.1.9/restore/Makefile	2020-05-06 20:53:33.474861013 +0200
@@ -114,9 +114,6 @@
 	$(INSTALL) -m 755 -d $(PKG_ROOT_SBIN_DIR)
 	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_ROOT_SBIN_DIR)
 	$(INSTALL) -m 755 -d $(PKG_SBIN_DIR)
-	# skip symlink when /sbin is alread symlinked to /usr/sbin, like on Fedora
-	test $(PKG_ROOT_SBIN_DIR) -ef $(PKG_SBIN_DIR) || \
-		$(INSTALL) -S $(PKG_ROOT_SBIN_DIR)/$(LTCOMMAND) $(PKG_SBIN_DIR)/$(LTCOMMAND)
 install-dev:
 
 .dep: $(COMMINCL) $(COMMON) $(INVINCL) $(INVCOMMON)
