diff -urN xfsdump-1.0.5.org/configure.in xfsdump-1.0.5/configure.in
--- xfsdump-1.0.5.org/configure.in	Tue May  1 22:18:41 2001
+++ xfsdump-1.0.5/configure.in	Tue May  1 22:56:12 2001
@@ -173,13 +176,6 @@
 dnl also check if man page source is gzipped
 dnl (usually on Debian, but not Redhat pre-7.0)
 have_zipped_manpages=false
-for d in ${prefix}/share/man ${prefix}/man ; do
-    if test -f $d/man1/man.1.gz
-    then
-	have_zipped_manpages=true
-	break
-    fi
-done
 AC_SUBST(have_zipped_manpages)
 
 dnl build definitions for use in Makefiles
